examples_tests/22.RaytracedAO/README.md

```diff
@@ -55,8 +55,10 @@ You can switch between those sensors using `PAGE UP/DOWN` Keys defined in more d
 |   bloomScale   | Denoiser Bloom Scale                                                                   | float  | 0.1                                                                                                                                                                      |
 | bloomIntensity | Denoiser Bloom Intensity                                                               | float  | 0.1                                                                                                                                                                      |
 |  bloomFilePath | Lens Flare File Path                                                                   | string | "../../media/kernels/physical_flare_512.exr"                                                                                                                             |
-|   tonemapper   | Tonemapper Settings for Denoiser                                                       | string | "ACES=0.4,0.8"                                                                                                                                                           |
-|   highQualityEdges   | Number in pixels (prevously was bool) to add to borders for more accurate denoising            | integer| 0                                                                                                                                                           |
+|   tonemapper   | Tonemapper Settings for Denoiser                                                       | string | "ACES=0.4,0.8"
+| cropOffsetX, cropOffsetY | Used to control the offset for cropping cubemap renders (instead of highQualityEdges)  | int    | 0                                                                                                                                                                        |
+| cropWidth, cropHeight    | Used to control the size for cropping cubemap renders (instead of highQualityEdges)    | int    | width-cropOffsetX, height-cropOffsetY                                                                      
+| envmapRegularizationFactor | if RIS is enabled then paths will be guided towards envmap based on this regularization factor.<br>1.0 means light paths will be distributed based on environmental map luminance.<br>0.0 means light paths will be distributed uniformly. | float  | 0.0                                                                                                                                                                      |               
```

Comment:
this still needs a wording fix

Reply:
Can you elaborate?
Tell me what needs to change.

Reply:
Done